lib/utils/repo-page-utils.ts

```diff
@@ -1,8 +1,45 @@
 import differenceInDays from "date-fns/differenceInDays";
 import { type StatsType } from "lib/hooks/api/useFetchMetricStats";
 
+export type HistogramData = {
+  bucket: string;
+  star_count?: number;
+  forks_count?: number;
+};
+
+export function getTicks({ histogram, range }: { histogram: HistogramData[]; range: number }): string[] {
```

Comment:
It'd be good to narrow range to th a narrower type, but that involves some changes outside this PR.

e.g. you could use https://github.com/open-sauced/app/blob/b92a95ec5900209dedd520bc87ff4d214faef2c2/components/shared/DayRangePicker.tsx#L4